test/1_api.js

```diff
@@ -119,6 +120,7 @@ describe('Server', function() {
 		storage.clear();
 		xhr = sinon.useFakeXMLHttpRequest();
 		clock = sinon.useFakeTimers();
+		server.createScript = function() {};
```

Comment:
To answer your question directly, I don't remember seeing it happened in the past, but I've seen instances of, when we try to stub `obj.func`, some browsers on saucelabs (I'm unable to reproduce it locally) will yell "attempted to stub undefined" on `obj.func`